Start enabling svelte builds

krisselden opened the pull request.

krisselden committed:
Start enabling svelte builds WIP

rwjblue commented on packages/@ember/deprecated-features/index.ts:
```diff
@@ -0,0 +1 @@
+export const DESCRIPTOR_VALUES = '3.2.0';
```
Perhaps we could go with `DESCRIPTOR_FALLBACK_PROPERTY_LOOKUP` or `PROPERTY_BASED_DESCRIPTORS`?

rwjblue reviewed (commented):
Minor quibble RE: deprecation name, but I don’t feel very strongly. Happy to have this land as is...

rwjblue commented:
Released an updated version of babel-plugin-debug-macros and submitted https://github.com/emberjs/ember.js/pull/16673 to update here (so we can test things out).

Commit:
Extract shared utility for requiring typescript.

Commit:
Use `!!'<version string>` for deprecated feature versions.

This ensures that other typescript files detect the import is (properly)
a boolean.

Commit:
Cleanup deprecate feature parsing.
